src/Utils.ts

```diff
@@ -5,18 +5,35 @@
  * @returns {string} - the randomly generated string.
  */
 function randomstring(length: number = 10) {
-    let result = '';
-    const randomchar = () => {
-        const n = Math.floor(Math.random() * 62);
-        if (n < 10)
-            return n; // 0-9
-        if (n < 36)
-            return String.fromCharCode(n + 55); // A-Z
-        return String.fromCharCode(n + 61); // a-z
-    };
-    while (result.length < length)
-        result += randomchar();
-    return result;
+  let result = '';
```

Comment:
compass.js is indented with 4 spaces. Please update your editor config.

I'll push a editorconfig in develop, which your IDE should use from now on.

Reply:
fixed formatting in Utils.ts